addons/mail/static/src/new/composer/composer_model.js

```diff
@@ -35,4 +35,15 @@ export class Composer {
             _store: store,
         });
     }
+
+    reset() {
+        Object.assign(this, {
+            textInputContent: "",
+            selection: {
+                start: 0,
+                end: 0,
+                direction: "none",
+            },
+        });
+    }
```

Comment:
There is only 2 assignations, I don't see why object.assign is useful here ?

```suggestion
    reset() {
            this.textInputContent = "";
            this.selection = {
                start: 0,
                end: 0,
                direction: "none",
            };
    }
```

Reply:
Was ignored in my push-forced. I just added follow-up commit in `master-discuss-refactoring` branch with this change: https://github.com/odoo-dev/odoo/commit/f215e1769b85f5e259190611c2790f68c39bf3d1